src/scalar_8x32_impl.h

```diff
@@ -179,9 +179,8 @@ static void secp256k1_scalar_cadd_bit(secp256k1_scalar *r, unsigned int bit, int
     r->d[7] = t & 0xFFFFFFFFULL;
 
     secp256k1_scalar_verify(r);
-#ifdef VERIFY
     VERIFY_CHECK((t >> 32) == 0);
-#endif
+    VERIFY_CHECK(secp256k1_scalar_check_overflow(r) == 0);
```

Comment:
33314ca: can we remove this since it's already done inside `secp256k1_scalar_verify`?

Reply:
This will be resolved by a rebase because it has already been done in #1184, see https://github.com/bitcoin-core/secp256k1/pull/1184#discussion_r1371803481.

Reply:
> [33314ca](https://github.com/bitcoin-core/secp256k1/commit/33314ca3b1dc55cdc4acf8f7327cfb57ba700101): can we remove this since it's already done inside `secp256k1_scalar_verify`?

Oops, no idea why I added this line in a commit which is supposed to only remove lines (and only add blank ones for better readability). Removed now.

> This will be resolved by a rebase because it has already been done in #1184, see [#1184 (comment)](https://github.com/bitcoin-core/secp256k1/pull/1184#discussion_r1371803481).



@real-or-random: The comment link refers to the same expression, but in a different function. The one in the review comment (being in `secp256k1_scalar_cadd_bit`) was indeed still there after the rebase.